capstone/src/main/webapp/index.html

```diff
@@ -7,23 +7,29 @@
     <script src="script.js"></script>
     <script src="explore-loader.js"></script>
   </head>
-  <body>
+  <body onload="getListings()">
     <div id="top-navigation"></div>
     <h1 id="explore-heading">Explore Clubs</h1>
     <div id="content">
-      <div id="club-listings" class="full-screen-panel">
-        <template id="club-listing">
-          <div class="club-logo-container">
-            <img src="" id="club-logo" class="club-logo" alt="Club logo">
-          </div>
-          <div class="club-info-container">
-            <h2><a href="" id="club-name"></a></h2>
-            <p id="description"></p>
-            <p id="members"></p>
-          </div> 
-          <form id="club-join-container" name="join" action="/student-data" method="POST"></form>
-        </template>
+      <div id="control-panel" class="full-screen-panel">
+        <select id="sort-type" onchange="getListings()">
+          <option value="default" selected>Default</option>
```

Comment:
I would lean towards naming this something that is more understandable to the user what this is sorted by. For example: "Recently added" or something similar. "Default" doesn't really give users much info about how this is sorted, might confuse the user what sort is happening. You can always add a comment to the logic to specify that the "recently added" is the default option.

Kind of like how most websites with search have a default but it's never name "default" in the list, usually it's something like "featured" or "relevancy". (Ex: Amazon's default is "featured").

WDYT?

Reply:
Yeah, I completely agree - I'm thinking about adding some additional sorting schemes, and for now I'll default to chronological (which is default, I'm just renaming). Maybe in the future I'll have some sort of "score" that I can rank by, by default.